<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dd::forms::FormSettingsWindow</class>
 <widget class="QWidget" name="dd::forms::FormSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EarBug Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="flyoutTab">
          <attribute name="title">
           <string>Flyout Position</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="flyoutGroupBox">
             <property name="title">
              <string>Fly Out</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="1" column="0">
                   <widget class="QPushButton" name="bottomLeftPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="FormMainWindow.qrc">
                      <normaloff>:/window_bottom_left.png</normaloff>:/window_bottom_left.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QPushButton" name="topLeftPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="FormMainWindow.qrc">
                      <normaloff>:/window_top_left.png</normaloff>:/window_top_left.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="topRightPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="FormMainWindow.qrc">
                      <normaloff>:/window_top_right.png</normaloff>:/window_top_right.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QPushButton" name="bottomRightPushButton">
                    <property name="maximumSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="FormMainWindow.qrc">
                      <normaloff>:/window_bottom_right.png</normaloff>:/window_bottom_right.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>128</width>
                      <height>128</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string>Offset</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>X Offset</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="xOffsetInput"/>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Y Offset</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="yOffsetInput"/>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="randomButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Keyboard</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Keyboard</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>467</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="FormMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
